game/hud/src/components/MOTD.tsx

```diff
@@ -26,55 +27,152 @@ export interface WelcomeStyles {
 }
```

Comment:
Because we're not using aphrodite anymore, you can get rid of this interface (WelcomeStyles) and remove the prop that is using it (styles)